rbac/common/addresser/family.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2018 Contributors to Hyperledger Sawtooth
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# -----------------------------------------------------------------------------
+
+from hashlib import sha512
+import re
+
+from rbac.common.base.base_family import BaseFamily
+
+
+class Family(BaseFamily):
```

Comment:
Yes, it's a design pattern.

Reply:
At this time, it is overengineering since there is only one case. Consider implementing inheritance if and when it is actually leveraged instead of frontloading it for something that might or might not exist.

Reply:
I am digressing on this per our discussion.